### ISSUE TITLE
Keep TOC accessible 

### ISSUE THREAD

**Issue by kelseiv** [influxdata/docs-v2#1556]
Keep TOC accessible while scrolling down a page on docs.influxdata.com.

Lack of accessibility especially evident on a long page like this:
https://docs.influxdata.com/enterprise_influxdb/v1.8/administration/cluster-commands/#show

**Comment by kelseiv** [comment]
@sanderson thoughts on time to fix? Does it make sense to resolve...or worth our while to let it be?

**Comment by sanderson** [comment]
This isn't a super simple fix and will have side effects:

- We have to make the left nav scrollable (separate from the content) for when it's too long to fit on the screen.
- This will likely change the current behavior of being able to collapse the left nav. How exactly, I'm not sure.
- Not a huge concern, but it will require some adjustments to how the left nav scales for mobile.

I myself prefer a nav that doesn't scroll independently, but understand why some prefer it this way. An alternative would be to add a "Back to top" button that follows users down the page. This would be simpler, but I don't know if it will appease those who prefer a "fixed" nav that follows them down the page.

**Comment by kelseiv** [comment]
@sanderson suggested adding a toggle option to give folks the option of having a scrolling nav versus a static nav.
